projects/vdk-plugins/vdk-structlog/tests/test_plugin.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pathlib
+from unittest import mock
+
+from click.testing import Result
+from vdk.plugin.structlog import plugin_entry
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
+
+"""
+This is a sample test file showing a recommended way to test new plugins.
+A good way to test a new plugin is how it would be used in the command that it extends.
+"""
+
+
+def test_dummy():
```

Comment:
I assume this test is unfinished, correct? I don't see how it's testing the plugin right now

Reply:
Yeah, it's unfinished. There's a separate ticket for e2e tests https://github.com/vmware/versatile-data-kit/issues/2803